Alphabetical order (#2350)
giantswarm/docs: src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md

The Cluster API controller that provisions the infrastructure in the vSphere environment needs a role with a set of permissions. To follow the principle of least privilege, it's recommended that a specific user and role be created for the controller.

> **Warning**: The password mustn’t contain ` \ ` (backslash) characters. Ideally restrict special characters to ` . , ! ? - `

Reviewer: ```suggestion
**Warning**: The password may not contain `\` (backslash) characters. Ideally restrict special characters to ` . , ! ? - `
```

("must not" isn't commonly known in non-natives)
